### DISCUSSION TITLE
Вывод

### DISCUSSION THREAD

**Discussion by lolyinseo** [shamhi/HamsterKombatBot#2396] [General]
Самое время обсудить вывод

![j49wd7xcnlod1 1](https://github.com/user-attachments/assets/2ba35505-1309-418b-be35-f64fa0b9c479)

**Comment by babasuck** [comment]
че думаете тем у кого ачивка за чит дадут хоть что-то или нет

**Reply by fractal-l** [reply]
ни

**Reply by lolyinseo** [reply]
> че думаете тем у кого ачивка за чит дадут хоть что-то или нет

Всем дадут. По опросу на реддите у 40% она. Многие не юзали никаких ботов, но ее получили

**Reply by Pipulas** [reply]
Интересно можно ли указать один кошелек у себя и жены, не охото регестрировать еще один акк на байбите. Или скажут я ботовод))

**Reply by lolyinseo** [reply]
> Интересно можно ли указать один кошелек у себя и жены, не охото регестрировать еще один акк на байбите. Или скажут я ботовод))

А ты как думаешь, почему такой лаг 21 - конец привязки, а дроп 26? За это время будут фильтровать скам. И не факт что одинаковые кошельки не проверят.

**Reply by Nior21** [reply]
Вот эта строка смущает, что она значит. Надо менять кошелек off-chain на что-то другое?
![image](https://github.com/user-attachments/assets/f0dd1e89-0788-4ddf-9a60-4358a77af11c)
Не понимаю тогда зачем его предлагали и почему прямым текстом нельзя сказать чтоб поменяли... Кто может подсказать:?
